Yoast/wordpress-seo
# Remove exclude from XML sitemap function for taxonomies

⚠️ This PR includes commits from another PR (#8830), and I'm not good enough with Git to remove them, so either merge that first or fix before merging. ⚠️ 

## Summary

This PR can be summarized in the following changelog entry:

* To bring parity with the other changes in 6.4, this introduces the question: "Allow search engines to show this `<taxonomy>` in search results?" and bases both the `noindex` and the inclusion in XML sitemaps on this decision.

## Test instructions

This PR can be tested by following these steps:

* Select "yes" on a category, see it included in XML sitemap and having no `noindex` meta value.
* Select "no" on a category, see it _not included in XML sitemap and _having_ a `noindex` meta value.

## Quality assurance


Fixes #2794
